.github/workflows/build_documentation.yml

```diff
@@ -2,8 +2,10 @@ name: Build documentation
 
 on:
   push:
-    tags:
-      - '[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
we should keep this as well no ?

Or do we need to make a new CI job for the releases that triggers on tags ?

Reply:
When we merge the release PR into main, that generates a push-on-main (with the right version), as well as merging this PR will trigger it for the "main" version).

Reply:
But the doc builder doesn't know it's a release no ? Since the tag is not merged with the PR

Reply:
I'm not sure of understanding the issue you raise.

I have reverted this file to its content before the commit 703b843. Before:
- whenever we merge a PR or commit directly to branch, a push-to-main event is triggered
- now I try to recover that behavior: all PR merges will trigger the push-to-main event

The doc builder creates the docs, depending on the version of the library:
- if it is a regular version, it generates the docs for that version
- of it is a ".dev" version, it generated the docs for the main branch

This way:
- when we merge this PR, the corresponding push-to-main event with ".dev" version should generate the docs for the main branch
- within the new release process, when we merge the release PR (see commit edf1902f954c5568daadebcd8754bdad44b02a85), the corresponding push-to-main event with regular version ("2.7.0" in the commit above) should generate the docs for the "2.7.0" version

Reply:
Ok I see ! Sorri I thought it would not have triggered the specific-version docs